Makefile

```diff
@@ -48,6 +49,12 @@ bin/atc: src/log/*.go src/atc/*.go
 	@mkdir -p bin
 	$(BUILD) -o $@ ${SRC}/atc
 
+jsx: src/react/jsx/*.js
+	@cd src/react && $(NPM) run build-js
```

Comment:
I think you probably want to log the `npm` command and not the `cd`. Putting the `@` at the beginning of the line suppresses logging for the entire rest of the line. Unfortunately I don't know a way to exec `npm` in the other directory without the `cd` on the same line.